index.js

```diff
@@ -0,0 +1,73 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+const patientGreetingService = require('./services/patient-greeting');
+
+const app = express();
+
+// This is necessary middleware to parse JSON into the incoming request body for POST requests
+app.use(bodyParser.json());
+
+app.use((request, response, next) => {
+  if (request.method !== 'GET' && request.headers && request.headers['content-type']
+    && request.headers['content-type'].match('json\\+fhir')) {
```

Comment:
We should remove this matching on `json+fhir`. This is a very old remnant from when CDS Hooks was using FHIR model for its API.